### PR TITLE
Update PSA jobs and use new way to enable it

### PR DESCRIPTION
**Pull request by xpivarc**
Update PSA lanes to 1.25 and use a new way to enable PSA rather than the dedicated provider.

### CODE REVIEW

**Comment by xpivarc** [comment]
/hold 
We need kubevirtci bump in kubevirt/kubevirt

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -1876,8 +1876,10 @@ presubmits:
             - -c
             - automation/test.sh
           env:
+            - name: KUBEVIRT_PSA
+              value: true
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:1880]: I think this needs to be `value: "true"`

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -2029,8 +2031,10 @@ presubmits:
         - -c
         - automation/test.sh
         env:
+        - name: KUBEVIRT_PSA
+          value: true
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2035]: Same as above - `value: "true"`

**Review by brianmcarey — COMMENTED** [review]
Looks good but failing the config validation

**Comment by xpivarc** [comment]
> Looks good but failing the config validation

Thanks!

**Review by brianmcarey — APPROVED** [review]
/lgtm
/approve

I think the hold can be removed once https://github.com/kubevirt/kubevirt/pull/8956 is merged.

**Pull request merged**